### DISCUSSION TITLE
Embed URL problem

### DISCUSSION THREAD

**Discussion by andrei-shershukov** [nextcord/nextcord#293] [Q&A] [answered]
I discover a problem with list of embeds, where if you have the same url in different embeds, only one of them will show up. Is it a discord problem or something wrong with nextcord embeds?
![Code_QooYqAbNoM](https://user-images.githubusercontent.com/72350904/146998440-a7319fe1-7ba5-4fd4-a03c-d73222a61c7f.png)
![Discord_xl0cuWzlXd](https://user-images.githubusercontent.com/72350904/146998460-c6e68a34-deac-42d2-802a-76a364b2c6b0.png)
![Code_w1fVxp4UKp](https://user-images.githubusercontent.com/72350904/146998473-5183211a-c8ba-4b25-9858-1dd44dba45f5.png)

**Comment by LeSpatiocorne** [comment]
Have you try to delay the send of the embeds ?
I mean tell to the bot to not send every embeds massively but for exemple send an embed every 1s up to 10 messages.

I say that because I'm pretty sure discord is limiting the message rate

**Comment by TAG-Epic** [comment, marked as answer]
Can't reproduce https://im.farfrom.world/Ik9BQ0tvMGxiMVF0VnVNMW5GR004MnFKUXdDWENVbVRIb2N6UGNidHkzcFl2UUEi0000YlGRig0000dXzIjGqlSivOrxusTE1dpt5uktA.png
